### PR TITLE
Add information about Wellcome Image Awards winners to descriptions

### LINKED ISSUE #1100: Expose Wellcome Image Awards field from MIRO in the API
From @jennpb:
Would it be possible to expose this MIRO field in the API?
Usage History > Wellcome Exhibitions > Exhibitions > e.g. Wellcome Image Awards 2012

It doesn’t have to be visible by 6 November, but it’d be nice if it could be by the end of the quarter.

### PR DESCRIPTION
**Pull request by alexwlchan**
### What is this PR trying to achieve?

Resolves #1100.

### Who is this change for?

API users who want to know about Award-Winning Images.

### Have the following been considered/are they needed?

- [ ] Deployed new versions

### CODE REVIEW

**Comment by alexwlchan** [comment]
This is a bit of a hack job; doing this in the truly general case is both dull and unnecessary.

File: common/src/main/scala/uk/ac/wellcome/models/transformable/miro/MiroTransformable.scala [hunk 1]
```diff
@@ -85,9 +85,50 @@ case class MiroTransformable(MiroID: String,
...
+      }
+
+      // Any more than two award-related entries in these fields would be
+      // unexpected, and we let it error as an unmatched case.
+    }
+
+    // Finally, remove any leading/trailing from the description, and drop
+    // the description if it's *only* whitespace.
     val description =
-      if (rawDescription.trim.length > 0) {
```
Comment (kenoir) [inline common/src/main/scala/uk/ac/wellcome/models/transformable/miro/MiroTransformable.scala]: `rawDescription.trim.isNotEmpty`

Comment (alexwlchan) [reply common/src/main/scala/uk/ac/wellcome/models/transformable/miro/MiroTransformable.scala]: 👌

**Comment by alexwlchan** [comment]
I think this is ready for another review. Once that’s done, we can get this merged and deployed ahead of the reindex on Monday.

**Pull request merged**